entries/b/bilibili.com.json

```
@@ -0,0 +1,12 @@
+{
+  "Bilibili": {
+    "domain": "bilibili.com",
+    "contact": {
+      "twitter": "bilibili_en",
+      "language": "en"
+    },
+    "keywords": [
+      "entertainment"
+    ]
```

As the website is entirely in Chinese I think specifying a regions element would be useful.
```suggestion
    ],
    "regions": [
      "cn"
    ]
```

---

The website is written in Chinese, but you can sign up from all over the world. In fact, Bilibili has many users in Japan. Language and region are two different concepts.

---

The regions element isn't meant to list regions where it's technically possible to use the site but the intended market of a site. If the creators of Bilibili had the ambition of reaching a global audience I'm pretty sure they would have added multi-language support.
E.g. [taobao.com](taobao.com) and [aliexpress.com](https://www.aliexpress.com/).